src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -210,12 +221,13 @@ const constructPageFetchUrl = ({
         fetchUrl = Url(`/${service}${variantPath}/topics/${id}`);
         break;
       }
+      // pathname is the ID of the Live page without /service/live/, and supports both Tipo & CPS IDs
```

Comment:
Can this comment be moved to above line 229 i.e. directly above the code which uses `pathname` (folk might miss this comment if it's outside the `case` statement